docs/developer/payments/transactions.mdx

```diff
@@ -407,22 +410,31 @@ Query variables:
 }
 ```
 
-:::info
-Enabling automatic checkout completion will not disrupt storefront functionality.
-If `checkoutComplete` is called for a `Checkout` that has already been converted to an `Order`,
-the corresponding `Order` instance will be return.
+:::danger
+**Enabling automatic checkout might disrupt storefront functionality!**
```

Comment:
this doc is not upgrade/migration guide. You can assume reader does not have any storefront here.

it's worth to mention this, but under the assumption somebody is actually migrating from the previous flow. Otherwise, I think it will be OK to mention a difference between enabled/disabled feature. You can represent it with a table or a list. 

For example:

```
One this setting is enabled, you should verify if your connected services, including storefront, can handle this flow. Please remember, that if setting is enabled:
- Checkout object will be removed automatically
- Running `checkoutComplete` on automatically completed checkout will return Order
- ...

```

Also it may be worth to describe what is the expected case on the storefront now. Previously storefront run `checkoutComplete` and received order.

Now it's not needed - but we may assume that storefront wants to show success message with the order. Do we recommend storefront to call `checkoutComplete` anyway, just to get access to order? Or maybe we should recommend some other query? Maybe we should add new api?

Reply:
Haven't we just found some DX issue with this api? That we need some better way to retrieve the order now?

Maybe we should follow up with something like `query Order(checkoutID: string)`?